### DISCUSSION TITLE
Related Elements not Pulling all Submissions

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#941] [Questions & Answers]
I have a few forms that relate to elements and having issues loading those elements on the front end.

In the cp I have the field in the element set up. Currently there are 4 entries from various forms.

The cp renders fine. All four entries are in the field. I can click on them and they take me to the submission.

if I do this on the front end it also works fine. There are four entries. 

```{% dd entry.notificationsAppointment.count() %}```

If I do this per freeforms docs things go wrong. It also does not work if I do a for loop of the entry field.

```{% set submissions = craft.freeform.submissions({ id: entry.notificationsAppointment.ids() }) %}```

OR

```{% for item in entry.notificationsAppointment.all() %} {{ item.id }} {% endfor %}```

I only get one entry. If I try to hard code the id line, I get the same issue... it only grabs one.

```id: [123, 345, 678]```

**Comment by kjmartens** [comment]
Hi @AbbeyDesign,

I'm just trying to wrap my head around what you're doing. Would you be able to share the full (though simplified if possible) template code you are attempting to use? 🙂 

And can you confirm which version of Craft and Freeform you're currently using?
